Community Toolbox/scripts/utils_CommunityToolboxDrawing/utils_CommunityToolboxDrawing.gml

```diff
@@ -0,0 +1,33 @@
+#region Settings
+
+/// @func draw_set_color_alpha(color,alpha)
+/// @desc Sets the drawing color and alpha.
+/// @arg {Constant.Color} color     The drawing color to set.
+/// @arg {Real} alpha               The alpha to set (between 0 and 1).
+function draw_set_color_alpha(_color, _alpha) {
+    gml_pragma("forceinline");
+    draw_set_color(_color);
+    draw_set_alpha(_alpha);
+}
+
+/// @func draw_set_colour_alpha(colour,alpha)
+/// @desc Sets the drawing colour and alpha.
+/// @arg {Constant.Color} colour    The drawing colour to set.
+/// @arg {Real} alpha               The alpha to set (between 0 and 1).
+function draw_set_colour_alpha(_colour, _alpha) {
+    gml_pragma("forceinline");
+    draw_set_colour(_colour);
```

Comment:
Love the attention to detail, calling the "correct" spelling of color/colour in each of these. :stuck_out_tongue: